be/src/olap/memtable.cpp

```diff
@@ -65,44 +75,46 @@ int MemTable::RowCursorComparator::operator()(const char* left, const char* righ
     return compare_row(lhs_row, rhs_row);
 }
 
-void MemTable::insert(const Tuple* tuple) {
+// For non-DUP models, for the data rows passed from the upper layer, when copying the data,
+// we first allocate from _buffer_mem_pool, and then check whether it already exists in
+// _skiplist.  If it exists, we aggregate the new row into the row in skiplist.
+// otherwise, we need to copy it into _table_mem_pool before we can insert it.
+void MemTable::_insert_agg(const Tuple* tuple) {
```

Comment:
we may change a name, such as `_insert_no_dup`? and put  `_insert_dup` before `_insert_no_dup`.
becasue this kind of `insert` agg and unique data.

Reply:
The unique table is essentially an Agg table where the aggregation function on the value column is replace, and the use of unique is only for the convenience of optimization.

Reply:
ok